junit/container/src/main/java/org/jboss/arquillian/junit/container/ContainerClassRulesFilter.java

```diff
@@ -0,0 +1,16 @@
+package org.jboss.arquillian.junit.container;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.jboss.arquillian.junit.JUnitClassRulesFilter;
+import org.junit.rules.TestRule;
+
+public class ContainerClassRulesFilter implements JUnitClassRulesFilter {
+
+    /**
+     * No @ClassRule should be executed inside of a container since the state between @Test is not kept there.
```

Comment:
Minor detail - might be worth expanding on what does it really mean that the state is not kept.